lite/examples/image_classification/ios/ImageClassification/ViewControllers/ViewController.swift

```diff
@@ -38,9 +38,13 @@ class ViewController: UIViewController {
   private var previousInferenceTimeMs: TimeInterval = Date.distantPast.timeIntervalSince1970 * 1000
 
 
+#if targetEnvironment(simulator)
```

Comment:
Same question on this file, too.
Could we separate this out into two files instead of using `#if` statements extensively?